basic.obo

```diff
@@ -219494,39 +220777,189 @@ created_by: http://orcid.org/0000-0001-9897-3238
+synonym: "liver lobule centrilobular region" RELATED [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+is_a: UBERON:8400005 ! metabolic zone of liver
 property_value: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0001172 ! hepatic acinus
 creation_date: 2020-09-08T15:05:03Z
 
 [Term]
 id: UBERON:8400003
-name: hepatic acinus intermediary zone
-def: "The region of the hepatic acinus in between the periportal zone and the perivenous zone. Blood in this region has an intermediate oxygen concentration relative to the other two zones of the hepatic acinus." [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
-synonym: "hepatic acinus zone 2" EXACT [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
-is_a: UBERON:0000063 ! organ subunit
-is_a: UBERON:0004119 ! endoderm-derived structure
+name: hepatic acinus zone 2
+def: "The region of the hepatic acinus in between zones 1 and 3. Blood in this region has an intermediate oxygen concentration relative to the other two zones of the hepatic acinus. Hepatocytes in this region have mixed functionality in comparison with those in zones 1 and 3." [http://www.ncbi.nlm.nih.gov/pubmed/29763190]
+synonym: "liver lobule midzonal region" RELATED [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+is_a: UBERON:8400005 ! metabolic zone of liver
 property_value: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0001172 ! hepatic acinus
 creation_date: 2020-09-08T15:15:57Z
 
+[Term]
+id: UBERON:8400005
+name: metabolic zone of liver
+def: "Any of the metabolic zones found in the hepatic acinus. Liver tissue is heterogenous at the level of morphometry and histochemistry. This heterogeneity is linked to the position of a cell within the functional unit of the tissue, which, in turn, is related to the blood supply: Cells located in the upstream zone differ from those in the downstream zone with respect to subcellular structures, key enzymes, translocators, and receptors and, therefore, have different metabolic capacities. This is the basis for \"metabolic zonation\", a concept first proposed for carbohydrate metabolism and later expanded to include amino acid and ammonia metabolism, xenobiotic metabolism, protective metabolism, and formation of plasma proteins." [http://www.ncbi.nlm.nih.gov/pubmed/8839925]
+synonym: "hepatic acinus metabolic zone" EXACT [http://www.ncbi.nlm.nih.gov/pubmed/8839925]
+is_a: UBERON:0004119 ! endoderm-derived structure
+property_value: dc-creator http://orcid.org/0000-0003-2034-601X
+creation_date: 2020-09-29T15:45:36Z
+
+[Term]
+id: UBERON:8400006
+name: liver lobule periportal region
+def: "The region of the liver lobule that is closest to the portal triad." [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
+synonym: "hepatic acinus zone 1" RELATED [http://www.ncbi.nlm.nih.gov/pubmed/28126520]
```

Comment:
Some confusion of acinus & lobule synonyms?  @mkeays

Reply:
@dosumis my understanding is that the acinus and its zones are functional concepts while the lobule and its zones are structural, though there is overlap between them and they are often used interchangeably -- the acinus zone 1 is generally found in the periportal zone of the lobule. [This comment](https://github.com/obophenotype/uberon/issues/1648#issuecomment-701433667) explains it. 
Do you think this would be better represented in a different way?

Reply:
Might be too confusing to have, for example, hepatic acinus zone 3 as a primary label and a related synonym on a different term - although I can understand why you'd do it given usage. Could we make an explicit relationship - e.g. overlaps and clarify relationship to usage in a comment?  Don't want to block release though, so maybe make a ticket.

Reply:
OK, sounds good. Have added #1790 to do this.